include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -1824,51 +1947,56 @@ namespace internal
               for (unsigned int i = 0;
                    i < transfer.schemes[0].n_dofs_per_cell_coarse;
                    i++)
-                level_dof_indices_fine_0[i] =
+                level_dof_indices_coarse[i] =
                   local_dof_indices[lexicographic_numbering_fine[i]];
 
               transfer.constraint_info_fine.read_dof_indices(
-                cell_no_0, level_dof_indices_fine_0, {});
+                cell_no[0], level_dof_indices_coarse, {});
             }
 
             // move pointers
             {
-              cell_no_0++;
+              cell_no[0]++;
```

Comment:
This is preexisting, but would we also want to use prefix increments,
```suggestion
              ++cell_no[0];
```